build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -848,11 +848,11 @@ message DirectoryNode {
   // The name of the directory.
   string name = 1;
 
-  // The digest of the
+  // The digest and optionally content of the
   // [Directory][build.bazel.remote.execution.v2.Directory] object
   // represented. See [Digest][build.bazel.remote.execution.v2.Digest]
   // for information about how to take the digest of a proto message.
-  Digest digest = 2;
+  Blob blob = 2;
```

Comment:
Is it permitted to also use this construct inside of Tree objects? If not, should it be stated? If so, any guidelines on canonicalisation?